docs/src/main/tut/chisel3/reset.md

```diff
@@ -11,10 +26,10 @@ with the register.
 
 There are three types of reset that implement a common trait `Reset`:
 * `Bool` - constructed with `Bool()`. Also known as "synchronous reset".
-* `AsyncReset` - constructed with `AsyncReset()`
-* `Reset`* - constructed with `Reset()`. Also known as "abstract reset".
+* `AsyncReset` - constructed with `AsyncReset()`. Also known as "asynchronous reset".
+* `Reset` - constructed with `Reset()`. Also known as "abstract reset".
 
-\* For implementation reasons, the concrete Scala type is `ResetType`. Stylistically we avoid `ResetType`, instead using the common trait `Reset`.
+For implementation reasons, the concrete Scala type is `ResetType`. Stylistically we avoid `ResetType`, instead using the common trait `Reset`.
```

Comment:
This wasn't stray, it was an intentional footnote style thing*

*I grant that it's maybe too close in the text to be obvious though.

Reply:
I don't think that renders well. Maybe actual footnotes work? I haven't tried that:

```
Hello [^foo]

[^foo]: Some footnote
```

Reply:
I see, it renders fine in on Github as markdown but I guess it doesn't work right on the website, well your change is fine then, I just hope it's clear enough that we're talking about the Scala type of what we above called `Reset`